Plugins/Trivia/TriviaPlugin.cs

```diff
@@ -96,29 +99,29 @@ public void Execute(PluginInput input)
                 }
                 else
                 {
-                    input.Callbacks.SendMessage("Eror! in order to start enter Start , in order to finish enter EXIT.");
+                    input.Callbacks.SendMessage(Resources.Plugins.Trivia_Eror1);
 
                 }
             }
         }
 
-        public string getTrivia()
+        public string GetTrivia()
         {
-            nums = rndNums(2, 10, true);
+            nums = RndNums(2, 10, true);
             result = nums[0] * nums[1];
-            allResult = rndNums(4, 100, false);
+            allResult = RndNums(4, 100, false);
             Random rnd = new Random();
             trueResult = rnd.Next(4);
             allResult.SetValue(result, trueResult);
-            var data = ($"what the result of: {nums[0]} * {nums[1]} =\n");
+            var data = ($"{Resources.Plugins.Trivia_Question} {nums[0]} * {nums[1]} =\n");
```

Comment:
The text in the resource should have the {}
For example
"what the result of: {0} * {1} =\n"
then do something like this
`string.format(Resources.Plugins.Trivia_Question, nums[0], nums[1])`

Reply:
This is how it should be written - string.Format(Resources.Plugins.Trivia_Question, nums[0], nums[1],"\n");
i performed.